runtime/vm/JVMImage.cpp

```diff
@@ -109,6 +109,11 @@ JVMImage::setupColdRun(void)
 		return IMAGE_ERROR;
 	}
 
+	_invalidTable = (J9ITable *) subAllocateMemory(sizeof(J9ITable));
```

Comment:
maybe also make _invalidTable->next = NULL;

Reply:
This might be wrong I haven't checked the struct

Reply:
I will take a look at this tomorrow